### DISCUSSION TITLE
[Feature] add different method for notification

### DISCUSSION THREAD

**Discussion by torik988** [immich-app/immich#11327] [Feature Request]
### I have searched the existing feature requests to make sure this is not a duplicate request.

- [X] Yes

### The feature

hey i wanted to know if it was possible to add to immich differents possibilities for admin notification, for example with critical storage or things like that.
Maby add an integration with discord webhook, ntfy, or other notification services

### Platform

- [X] Server
- [ ] Web
- [ ] Mobile

**Comment by bo0tzz** [comment]
System level notifications like storage filling up should be handled by monitoring tools, not by Immich.
